source/simulator/core.cc

```diff
@@ -1043,6 +1046,7 @@ namespace aspect
             {
               coupling[x.velocities[d]][x.pressure] = DoFTools::always;
               coupling[x.pressure][x.velocities[d]] = DoFTools::always;
+              coupling[x.pressure][x.pressure] = DoFTools::always;
```

Comment:
> tried to prescribe pressures

Interesting. I don't like having to add coupling for the pressures but I think this should work without. Do you really need to prescribe pressures? This is not something typically done for the Stokes problem.